metadata.c

```diff
@@ -403,15 +403,15 @@ static int pv_usermeta_parse(struct pantavisor *pv, char *buf)
 		n = (*key_i)->end - (*key_i)->start;
 
 		// copy key
-		key = calloc(1, (n+1) + sizeof(char*));
+    key = calloc(n + 1 + 1, sizeof(char));
```

Comment:
is this 1+1 needed=?

Reply:
also: indent

Reply:
@anibalportero my bad, yes looking at the code `+ 1` is enough my bad
@asac I will fix it

Reply:
Fixed in 95ec83b7d8abe1e192ecc840cdfc54a6069f59dd